threads.php

```diff
@@ -375,3 +380,58 @@ function cfth_timeline_css() {
 <?php
 }
 
+// Meta field markup for add thread page
+function threads_taxonomy_add_new_meta_field() {
+	?>
+	<div class="form-field">
+		<label for="term_meta[order_term_meta]"><?php _e( 'Post order', 'threads' ); ?></label>
+			<select name="term_meta[order_term_meta]" id="term_meta[order_term_meta]">
+				<option value="ASC"><?php _e( 'ASC', 'threads' ); ?></option>
+				<option value="DESC"><?php _e( 'DESC', 'threads' ); ?></option>
+			</select>
+		<p class="description"><?php _e( 'Enter a value for this field','threads' ); ?></p>
+	</div>
+<?php
+}
+
+add_action( 'threads_add_form_fields', 'threads_taxonomy_add_new_meta_field', 10, 2 );
+
+// Meta field markup for edit thread page
+function threads_taxonomy_edit_meta_field( $term ) {
+	$term_meta = get_term_meta( $term->term_id, 'threads_meta', true ); 
+	?>
+
+	<tr class="form-field">
+		<th scope="row" valign="top"><label for="term_meta[order_term_meta]"><?php _e( 'Post order', 'threads' ); ?></label></th>
+		<td>
+			<select name="term_meta[order_term_meta]" id="term_meta[order_term_meta]">
+				<option value="ASC" <?php echo ( isset( $term_meta['order_term_meta'] ) && 'ASC' === $term_meta['order_term_meta'] )? 'selected' : ''; ?> ><?php _e( 'ASC', 'threads' ); ?></option>
+				<option value="DESC" <?php echo ( isset( $term_meta['order_term_meta'] ) && 'DESC' === $term_meta['order_term_meta'] )? 'selected' : ''; ?>><?php _e( 'DESC', 'threads' ); ?></option>
+			</select>
+			<p class="description"><?php _e( 'Enter a value for this field','threads' ); ?></p>
```

Comment:
This description is also very meaningless. Instead, it should probably read "Sort threaded posts with newest first (DESC), or oldest first (ASC)."

You could also probably change the select input option names to be more humanized as well:
"Oldest to Newest (ASC)" and "Newest to Oldest (ASC)"